rust+wasm/init-msg.rhai

```diff
@@ -0,0 +1,10 @@
+print();
+if variable::get("is_init") {
+    print("\
+        Remember, if the generator detects a directory or file conflict, it will not alter your project,\n\
+        so carefully read the prompts.");
```

Comment:
```suggestion
        If the generator detects a directory or file that already exists, it will not alter your project,\n\
        so read the prompts carefully.");
```